content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+author:      "GeoB99"
+date:        2023-08-06
+tags:        [ newsletter ]
+---
+
+Salutations from the ReactOS project team! On the previous newsletter we talked about the overall ReactOS releasing progress scheme and development status of the project.
+We are attempting to make an effort on publishing regularly at least 3 newsletters per year, depending on how the development workflow goes.
+In this newsletter we will highlight some of the contributions by project developers and contributors alike as well as future plans and headlines.
+
+## 0.4.15 release status
+On [102nd newsletter](/blogs/newsletter-102/) we talked about the ReactOS releasing situation and how we are tackling it in these current times.
+Despite our on-going effort and continuation for the project's development, lots of people have asked about what happened to 0.4.15 and if there will be such a release.
+The answer to these questions is yes -- there will be 0.4.15 and it will be released. The 0.4.15 release has been long time delayed due to outstanding regressions and bugs.
+
+Most notably it was ([CORE-17624](https://jira.reactos.org/browse/CORE-17624)) that was long unfixed for some time after the soft rewrite of the Memory manager.
+While it has been hackfixed by ([#5630](https://github.com/reactos/reactos/pull/5630)) as a proper solution is yet to be worked on, it is one step closer to branch 0.4.15 for a candidate release.
+There are still other minor regressions that need to be ironed out until 0.4.15 reaches the final stage for a release.
+
+Due to development resources and manpower constraints we cannot give an exact estimation when 0.4.15 will be released to the general public but we will eventually get there. We are sorry for the inconvenience!
+In the meantime you can try [nightly builds](https://reactos.org/getbuilds) that pack up with latest bugfixes, new features and whatnot but expect also new bugs and such!
+
+## Kernel Debugger overhaul
+Historically, ReactOS was built with the GCC compiler, and to help with kernel development an in-built local kernel debugger, KDBG, was added. This debugger possesses its own set of commands and syntax, can understand only an in-house debugging-symbols format (“rossym”) – it does not support PDB. However, being a local debugger, it is able to interact with the user via a serial port connection or locally via screen display; input can be done either via serial or via keyboard (PS/2).
+Later on, the possibility to compile ReactOS with MSVC was added, and with it too, support for the WinDbg debugger. Kernel debugging with WinDbg works by having a debugging stub (KD64, this is a similar idea to GDB) on the target, that is capable of manipulating the state of the OS, and having the actual debugger offloaded to a host computer.
+
+Hermès is working on improving the ReactOS' kernel debugger (KDBG) with new features and miscellaneous fixes so that it makes easier for developers and testers alike to reliably debug ReactOS.
```

Comment:
```suggestion
Hermès BÉLUSCA - MAÏTO (HBelusca) is working on improving the ReactOS' kernel debugger (KDBG) with new features and miscellaneous fixes so that it makes easier for developers and testers alike to reliably debug ReactOS.
```

Since this is the first time mentioning Hermès